runtime/str.go

```diff
@@ -157,6 +157,29 @@ func strAdd(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return NewStr(stringV + stringW).ToObject(), nil
 }
 
+func strCapitalize(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "capitalize", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	buf := bytes.Buffer{}
+	runes := []rune(s)
+	numRunes := len(runes)
+	writeRune := func(r rune, transform func(r rune) rune) {
+		if r <= unicode.MaxASCII {
+			r = transform(r)
+		}
+		buf.WriteRune(r)
+	}
+	if numRunes > 0 {
+		writeRune(runes[0], func(r rune) rune { return unicode.ToUpper(r) })
+	}
+	for i := 1; i < numRunes; i++ {
```

Comment:
Hmmm, after rereading what I have is way too complicated.  How about something like the following? :

```
s := toStrUnsafe(args[0])
sv := s.Value()
if len(sv) == 0 {
        return s.ToObject(), nil
}
return NewStr(strings.ToUpper(sv[0:1]) + strings.ToLower(sv[1:])).ToObject(), nil
```

It is more in line with `str.lower`, `str.upper`, etc...